### PR TITLE
Chrome Extension Build

### PR DESCRIPTION
**Pull request by PradyumnaKrishna**
Hey, I brought some changes here.

With the help of `manifest.json` anyone can load it as chrome extension. I have added a `M` icon to this with gradient on and off button which gives people a choice.

Check it out and if you like it then merge this pull request.

### CODE REVIEW

**Review by boriskrasko — APPROVED** [review]

**Review by boriskrasko — APPROVED** [review]

**Review by boriskrasko — APPROVED** [review]

File: main.js [hunk 1]
```diff
@@ -170,13 +171,25 @@ const cleanStorage= () => {
...
   focus.textContent = '[Enter Focus]'
-  city.textContent = '[Enter City]'
+  city.textContent = 'Earth'
   curtainBackground.style.filter = 'blur(10px)'
   grayOrColorIcon.src = 'svg/brightness.svg'
   document.body.style.overflow = 'hidden'
   setTimeout(backgroundFilter, 500)
 }
 
+const setGradient= () => {
```
Comment (boriskrasko) [inline main.js:181]: Good idea, friend. But I think it's not entirely justified to include the jQuery library for one function only. I would write this function a native JavaScript.
``` javascript
const setGradient = () => {
  greeting.classList.toggle('gradient-font');
  name.classList.toggle('gradient-font');
  gradient.classList.toggle('active');
}
```

**Review by boriskrasko — COMMENTED** [review]
Good idea, friend. But I think it's not entirely justified to include the jQuery library for one function only. I would write this function a native JavaScript.
``` javascript
const setGradient = () => {
  greeting.classList.toggle('gradient-font');
  name.classList.toggle('gradient-font');
  gradient.classList.toggle('active');
}
```

**Review by boriskrasko — APPROVED** [review]

File: main.js (hunk 1 above)
Comment (PradyumnaKrishna) [reply main.js:181]: I am not a pro in js and thanks for giving a good alternative.
I has removed jquery as per your request

**Review by boriskrasko — APPROVED** [review]
Awesome work

**Pull request merged**